scaling.adoc

```diff
@@ -243,92 +243,92 @@ different versions of Bitcoin Core going back to version 0.8.
+3452 minutes. All of this - roughly 10x - improvement is due to inward scaling.
 
-The improvements could be categorized as either space (RAM, disk,
-bandwidth, etc.) savings or computational savings. Both categories
+The improvements could be categorized as either saving space (RAM, disk,
+bandwidth, etc.) or saving computational power. Both categories
 contribute to the improvements in the diagram above.
 
-A good example of computational improvements can be found in the
-https://github.com/bitcoin-core/secp256k1[libsecp256k1] library, which
+A good example of computational improvement can be found in the
+https://github.com/bitcoin-core/secp256k1[libsecp256k1] library, which,
 among other things, implements the cryptographic primitives needed to
 make and verify digital signatures. Pieter Wuille is one of the
-contributors to this library, and he
-https://twitter.com/pwuille/status/1450471673321381896[wrote a twitter
-thread] showcasing the performance improvements made by various pull
+contributors to this library, and he wrote a
+https://twitter.com/pwuille/status/1450471673321381896[Twitter
+thread] showcasing the performance improvements achieved through various pull
 requests.
 
 .Performance of signature verification over time, with significant pull requests marked on the timeline. Source: https://twitter.com/pwuille/status/1450471673321381896
 image::libsecp256k1speedups.png[{half-width}]
 
-The graph shows the trend for two different 64-bit CPU types, ARM and x86.
+The graph shows the trend for two different 64-bit CPU types, namely ARM and x86.
 The difference in performance is due to the more specialized instructions
-available on x86 compared to the ARM architecture, which has fewer,
-more generic instructions. But the general trend is the same for both
+available on x86 compared to the ARM architecture, which has fewer and
+more generic instructions. However, the general trend is the same for both
 architectures. Note that the Y-axis is logarithmic, which makes the
 improvements look less impressive than they actually are.
 
-There are also several good examples of space savings contributing to
-performance improvements. In a
+There are also several good examples of space-saving improvements that contributed to
+performance enhancement. In a
 https://murchandamus.medium.com/2-of-3-multisig-inputs-using-pay-to-taproot-d5faf2312ba3[Medium
-blog post] about Taproot's contribution to space savings, user Murch
-compared how much block space a 2-of-3 threshold signature would
-require, both without using Taproot and using Taproot in various ways.
+blog post] about Taproot's contribution to saving space, user Murch
+compares how much block space a 2-of-3 threshold signature would
+require, using Taproot in various ways as well as not using it at all.
 
-.Space savings for different spending types Taproot and legacy versions.
+.Space savings for different spending types, Taproot and Legacy versions.
```

Comment:
Why Legacy with captial "L"? I use the word "legacy" to refer to Bitcoin payments made using tech available prior to Taproot.

Reply:
Yeah, me too. I have found it with capital L in the past, but you are right in noticing that there is no Bitcoin upgrade called "Legacy". I'll change it back.